Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -175,6 +175,7 @@
 pathALCARECOSiStripCalMinBiasAAG = cms.Path(seqALCARECOSiStripCalMinBiasAAG*ALCARECOSiStripCalMinBiasAAGDQM)
 pathALCARECOSiStripCalSmallBiasScan = cms.Path(seqALCARECOSiStripCalSmallBiasScan)
 pathALCARECOSiStripCalZeroBias = cms.Path(seqALCARECOSiStripCalZeroBias*ALCARECOSiStripCalZeroBiasDQM)
+#pathALCARECOSiStripCalZeroBias.replace(consecutiveHEs, unpackTcds*consecutiveHEs)
```

Comment:
@vischia if this no more needed, please remove the commented code.

Reply:
Fixed